src/app/styles/partials/_base.scss

```diff
@@ -39,3 +39,9 @@ $border-radius: 10px;
 //margin
 $margin-med: 10px;
 $margin-large: 20px;
+
+//game-wrapper component margin
+$margin-bottom: 100px;
```

Comment:
Same with this, keep the names generic. You might just not use 100px margin for bottom, or left.

Reply:
Done.